python/kserve/kserve/model_server.py

```diff
@@ -124,7 +129,9 @@ def start(self, models: Union[List[Model], Dict[str, Deployment]]) -> None:
                 else:
                     raise RuntimeError("Model type should be 'Model'")
         elif isinstance(models, dict):
-            if all([isinstance(v, Deployment) for v in models.values()]):
+            from ray import serve as rayserve
+
+            if all([has_rayserve and isinstance(v, Deployment) for v in models.values()]):
```

Comment:
probably it is not a good idea to check the flag and let it silently fail, we'd rather have a hard failure if rayserve is used but not installed as a dependency.

Reply:
It will fail explicitly as we are importing rayserve without any exception handling. It is just a redundant check i will remove it

Reply:
removed the redundant check